draft-ietf-dnsop-dns-catalog-zones.md

```diff
@@ -197,30 +199,34 @@ Nameservers MUST accept catalog zones even with those labels not really unique;
 Having a large number of member zones in a single RRset may cause the RRset to be too large to be conveyed via DNS messages which make up a zone transfer.
 Having the zones uniquely tagged with the `<m-unique-N>` label ensures the list of member zones can be split over multiple DNS messages in a zone transfer.
 
-The `<m-unique-N>` label also enables the state for a zone to be reset. (see (#zonereset))
+The `<m-unique-N>` label also enables the state for a zone to be reset (see (#zonereset)).
 As long as no zone state needs to be reset at the authoritative nameservers, the unique label associated with a zone SHOULD remain the same.
 
 The CLASS field of every RR in a catalog zone MUST be IN (1).
 
+[FIXME: how about turning this around? Because they are not for querying, queries to it should be discouraged by using a huge TTL]
 The TTL field's value is not specially defined by this memo.  Catalog zones are
 for authoritative nameserver management only and are not intended for general
 querying via recursive resolvers and therefore a value of zero (0) is
 RECOMMENDED.
 
-Each RRSet of catalog zone, with the exception of the zone apex, SHOULD consist of just one RR. It's acceptable to generate owner names with the help of a
+[FIXME: earlier we said that the version RRset could have multiple entries]
+Each RRSet of a catalog zone, with the exception of the zone apex, SHOULD consist of just one RR. It's acceptable to generate owner names with the help of a
```

Comment:
This will limit future implementation/operator specific extensions. If they need more than one record per rrset to reach there goal this should be possible.
I think we should limit the number of records in a rrset only on a per property basis, when needed.

Reply:
This was to prevent creation of very large RRsets, because large RRsets don't work too well in transfers. The zones could for example have been a single RRset of PTR records. (if we would not have properties).

How about leaving it for now, but allowing for exceptions, like:
"Unless indicated otherwise per case basis, each RRsets of a catalog zone SHOULD consist of just one RR"

Reply:
That works, but we could also be explicit per property. A zone has up to one serial. A zone has up to one name. A zone belongs to up to one group. A zone has zero or more name servers (not a property we have now, but as an example).

This will set a good example for future property definitions to also be explicit.

Reply:
I removed the text and we can specify it per property.